{"uuid":"1126a8ff-77a8-4a15-8480-6307f36004fc","historyId":"826f701409236b4f4d9ff5305dd1ced7","fullName":"org.example.ex05_Selenium_Waits.TestSelenium23_Fix_Miniproject.test_negative_vwo_login","labels":[{"name":"package","value":"org.example.ex05_Selenium_Waits.TestSelenium23_Fix_Miniproject"},{"name":"testClass","value":"org.example.ex05_Selenium_Waits.TestSelenium23_Fix_Miniproject"},{"name":"testMethod","value":"test_negative_vwo_login"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"LearningSelenium_ATB10x"},{"name":"subSuite","value":"org.example.ex05_Selenium_Waits.TestSelenium23_Fix_Miniproject"},{"name":"host","value":"LAPTOP-AK770PPM"},{"name":"thread","value":"15400@LAPTOP-AK770PPM.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"test_negative_vwo_login","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [your email, password, IP address or location did not match] but found [Your email, password, IP address or location did not match]","trace":"java.lang.AssertionError: expected [your email, password, IP address or location did not match] but found [Your email, password, IP address or location did not match]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1590)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat org.example.ex05_Selenium_Waits.TestSelenium23_Fix_Miniproject.test_negative_vwo_login(TestSelenium23_Fix_Miniproject.java:83)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Verify that with invalid email, pass, error message is shown on the app.vwo.com","steps":[],"attachments":[],"parameters":[],"start":1743703723643,"stop":1743703739120}